test/refactoring/ConvertAsync/basic.swift

```diff
@@ -0,0 +1,187 @@
+enum CustomError: Error {
+  case invalid
+  case insecure
+}
+
+typealias SomeCallback = (String) -> Void
+typealias SomeResultCallback = (Result<String, CustomError>) -> Void
+typealias NestedAliasCallback = SomeCallback
+
+// 1. Check that the next functions are treated as having async alternatives
+
+// RUN: %refactor -add-async-alternative -dump-text -source-filename %s -pos=%(line+4):1 | %FileCheck -check-prefix=ASYNC-SIMPLE %s
+// RUN: %refactor -add-async-alternative -dump-text -source-filename %s -pos=%(line+3):6 | %FileCheck -check-prefix=ASYNC-SIMPLE %s
+// RUN: %refactor -add-async-alternative -dump-text -source-filename %s -pos=%(line+2):12 | %FileCheck -check-prefix=ASYNC-SIMPLE %s
+// RUN: %refactor -add-async-alternative -dump-text -source-filename %s -pos=%(line+1):13 | %FileCheck -check-prefix=ASYNC-SIMPLE %s
+func simple(completion: (String) -> Void) { }
+// ASYNC-SIMPLE: basic.swift [[# @LINE-1]]:1 -> [[# @LINE-1]]:1
+// ASYNC-SIMPLE-NEXT: @available(*, deprecated, message: "Prefer async alternative instead")
+// ASYNC-SIMPLE: basic.swift [[# @LINE-3]]:46 -> [[# @LINE-3]]:46
+// ASYNC-SIMPLE-EMPTY:
+// ASYNC-SIMPLE-EMPTY:
```

Comment:
How many new lines are added by the replacement printing versus in the replacement string itself? Just wondering if two empty lines mean a single newline in the replacement or two.

Reply:
That's... a good point. It's two newlines in the replacement string, which gets printed as two lines in the printing. Kind of weird but not sure what else we'd do there.